bitnami/harbor/templates/trivy/trivy-sts.yaml

```diff
@@ -57,7 +57,7 @@ spec:
       {{- if .Values.podSecurityContext }}
       securityContext: {{- include "common.tplvalues.render" (dict "value" .Values.podSecurityContext "context" $) | nindent 8 }}
       {{- end }}
-      automountServiceAccountToken: {{ .Values.trivy.automountServiceAccountToken }}
+      automountServiceAccountToken: {{ .Values.trivy.automountServiceAccountToken | default false }}
```

Comment:
The default should come from the values.yaml so the pipe is not needed. This is repeated in similar lines.